sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_connection.py

```diff
@@ -817,20 +836,27 @@ def create_session(self, **kwargs):
         :rtype: ~pyamqp._session.Session
         """
         assigned_channel = self._get_next_outgoing_channel()
-        kwargs["allow_pipelined_open"] = self._allow_pipelined_open
-        kwargs["idle_wait_time"] = self._idle_wait_time
+        kwargs['offered_capabilities'] = offered_capabilities or self._offered_capabilities
```

Comment:
I don't think we should be populating the Connection-level `offered_capabilities` with the Session-level `offered_capabilities`?